tests/unit/test_transform.py

```diff
@@ -59,6 +59,18 @@ def test_mask_hidden(self):
             'hidden'
         )
 
+    def test_mask_skip_ends(self):
```

Comment:
can you add couple tests for usecases where value to mask is not long enough?

Reply:
Will add those.